test/verify/pytest_dynamic_loader.py

```diff
@@ -78,31 +78,25 @@ def identifier(text):
     return "/" + text
 
 
+def default_key(parent_dict, key, empty_obj):
```

Comment:
Please squash this "lets simplify fmf formatter function" commit into the previous commit.